src/shared/logger/activation.ts

```diff
@@ -154,13 +153,17 @@ function getLogLevel(): LogLevel {
 }
 
 function makeLogFilename(): string {
-    const m = moment()
-    const date = m.format('YYYYMMDD')
-    const time = m.format('HHmmss')
-    // the 'T' matches VS Code's log file name format
-    const datetime = `${date}T${time}`
-
-    return `aws_toolkit_${datetime}.log`
+    const now = new Date()
+    // local to machine: use getMonth/Date instead of UTC equivalent
+    // month is offset by 1
+    const m = now.getMonth() + 1 < 10 ? `0${now.getMonth() + 1}` : now.getMonth() + 1
+    const d = now.getDate() < 10 ? `0${now.getDate()}` : now.getDate()
+    const h = now.getHours() < 10 ? `0${now.getHours()}` : now.getHours()
+    const mn = now.getMinutes() < 10 ? `0${now.getMinutes()}` : now.getMinutes()
+    const s = now.getSeconds() < 10 ? `0${now.getSeconds()}` : now.getSeconds()
```

Comment:
`padStart(2, '0')` can be used to ensure 2-digit number.

```suggestion
    const m = (now.getMonth() + 1).toString().padStart(2, '0')
    const d = now.getDate().toString().padStart(2, '0')
    const h = now.getHours().toString().padStart(2, '0')
    const mn = now.getMinutes().toString().padStart(2, '0')
    const s = now.getSeconds().toString().padStart(2, '0')
```

Reply:
can't we use `toISOString()` or something?

Reply:
Do we want that in UTC? I think this one's typically local.

Reply:
UTC sounds completely fine for a logfile name.